app/assets/stylesheets/bootstrap/bootstrap-theme.css

```diff
@@ -0,0 +1,587 @@
+/*!
+ * Bootstrap v3.3.6 (http://getbootstrap.com)
```

Comment:
Put third-party libraries like bootstrap in vendor/assets/stylesheets to keep them out of your "app" space. Or use the gem versions to avoid the whole issue.